docs/Versions.asciidoc

```diff
@@ -9,7 +9,7 @@
 release-state can be: released | prerelease | unreleased
 //////////
 
-:release-state:   prerelease
+:release-state:   unreleased
```

Comment:
I had to make these two changes otherwise the links are incorrect on current master. I suspect this has to do with the fact that we don't have an alpha2 branch hence we have been building alpha2 docs from master. This change may break alpha2 docs then, not sure.